packages/redux-devtools-app/src/reducers/theme.ts

```diff
@@ -21,7 +25,20 @@ export default function theme(
       theme: action.theme,
       scheme: action.scheme,
       light: !action.dark,
+      latestChangeBy: CHANGE_THEME,
     };
   }
+
+  if (
+    action.type === APPLY_MEDIA_FEATURES_PREFERENCES &&
+    state.latestChangeBy !== CHANGE_THEME
+  ) {
```

Comment:
Instead of checking on startup can we make it three options: "use system default", "light", or "dark"?

Reply:
sure but we have got to store this preference in a different property name or it would not work properly due to the way the store data is hydrated.

Nice idea